3.4/arm/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+# moodledb is the database host
+moodledb_rpi:
+  image: arm32v7/postgres
+  container_name: moodledb_rpi
+  ports:
+    - "5432:5432"
+  environment:
+    - POSTGRES_DATABASE=moodle
+    - POSTGRES_USER=moodle
+    - POSTGRES_PASSWORD=moodle
+moodle_rpi:
+  build: .
+  container_name: moodle_rpi
+  ports:
+    - "8080:80"
+  links:
+    - moodledb_rpi
+  environment:
+    - MOODLE_URL=http://docker.ole.org:80
+  volumes:
+    - ./config/php.ini:/etc/php/7.0/apache2/php.ini
```

Comment:
could you tell me what this one for? why share the volume? can we delete the config as we have in x86?